### DISCUSSION TITLE
kendala LazyVim

### DISCUSSION THREAD

**Discussion by Rauliqbal** [deaafrizal/forum#69] [Troubleshooting Room]
![Screenshot from 2024-02-20 14-38-07](https://github.com/deaaprizal/forum/assets/89615372/a4096051-07ec-41d3-b579-47dc0bb4c196)

kenapa terminal nya begini bang ?

**Solved**

**Comment by rexyb9** [comment]
mungkin itu bentrok sama zshnya

**Reply by Rauliqbal** [reply]
owh begitu...makasih yaa mas

**Reply by deaafrizal** [reply]
bukanya pake space + f + t atau key lain bro?

**Reply by Rauliqbal** [reply]
entah deh, soalnya kan config dari bang dea

**Comment by Alfadjri** [comment]
Mungkin bisa membantu 

Aku pakek iTerm settingan nyontek dikit dari bang dea  kalau yang terminalnya jadi floating kaya di gambar bisa pakek plugin akinsho/toggleterm.nvim

ini link github nya https://github.com/akinsho/toggleterm.nvim
itu settingan di  lazy.lua
![gambar 1](https://github.com/deaaprizal/forum/assets/56843629/c4904f09-4980-488c-af5a-b46dbaf520d8)





terus buat file baru di plugin namanya toggleterm.lua
<img width="1680" alt="Screenshot 2024-04-08 at 20 59 25" src="https://github.com/deaaprizal/forum/assets/56843629/7952b378-4ec4-421d-b56a-bae909216521">

tinggal ganti aja key nya nnti muncul tu di settingan 
kalau di gambar itu aku timban default key dari lazyVim nya jadi ft kalau fT nnti floating lagi 

ini hasil akhirnya 
![Screenshot 2024-04-08 at 21 04 38](https://github.com/deaaprizal/forum/assets/56843629/3eb9b052-b4db-4b1e-854c-b9cf9529944e)


Semoga membantu

**Reply by Rauliqbal** [reply]
Thanks bang,, solved

**Comment by Dapow221** [comment]
<img width="1440" alt="Screenshot 2024-04-24 at 22 08 02" src="https://github.com/deaaprizal/forum/assets/141142350/4ea1fca2-4f34-42ed-8076-724388d2384d">

bang punya saya kok begini yahh apakah ada solusi?? udah googling jugak

**Reply by Rauliqbal** [reply]
Itu folder nvim di .config , harus di hapus dulu

**Comment by Alfadjri** [comment]
itu harus di overwrite dulu cara nya itu download dulu nvim bang dea di
tempat lain terus di mv  pakek perintah mv ... ~/.config/nvim kalau
misalnya masih gagal coba pakek sudo dulu

**Reply by Rauliqbal** [reply]
Udah fix bang, ternyata install togleterm
